code/game/machinery/quantum_pad.dm

```diff
@@ -190,6 +191,23 @@
 		linked_pad = link
 		. = TRUE
 
+/obj/machinery/quantumpad/maintenance
+	name = "old quantum pad"
+	desc = "A mysterious old quantum pad that fits underneath the floor. Where does it lead?"
+	sound_volume = 5
+	teleport_cooldown = 5 SECONDS //gives you a head start on the HoS
+	teleport_speed = 0.5 SECONDS //somehow faster than new ones despite only having base parts
+	use_power = NO_POWER_USE //would make more sense if it had a replaceable battery but it doesn't
+	flags_1 = NODECONSTRUCT_1 //prevent breaking it unintentionally or out of curiousity
+
+/obj/machinery/quantumpad/maintenance/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/undertile, TRAIT_T_RAY_VISIBLE, INVISIBILITY_OBSERVER)
```

Comment:
This stopped working properly. The pads start above the floor and are only hidden when you remove and replace the tile. Don't know how to fix it.